index.html

```diff
@@ -1,14 +1,78 @@
 <!DOCTYPE html>
 <html>
- <head>
-  <meta charset="UTF-8">
-  <title>title</title>
-  <meta name="author" content="name">
-  <meta name="description" content="description here">
-  <meta name="keywords" content="keywords,here">
-  <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
- </head>
- <body>
-
- </body>
-</html>
\ No newline at end of file
+<head>
+	<meta charset="UTF-8">
+	<title>Kevin Smith Transportation Group</title>
+	<meta name="description" content="Philadelphia region's newest luxury limoursine and total transportation service.">
+	<meta name="keywords" content="luxury, limo, transportation, Philadelphia">
+	<link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+	<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,400,700' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="css/reset.css">
+	<link rel="stylesheet" type="text/css" href="css/stylesheet.css">
+</head>
+<body>
+	<header>
+		<div>
+			<img src="images/ktsg-logo.png" alt="Kevin Smith Transportation Group" height="150" width="288">
+		</div>
+	</header>
+	<section>
+		<h1>Your Luxury Limousine and Total Transportation Service.<br />Arriving July 2015.</h1>
```

Comment:
I would probably get rid of the `<br />` and make 'Arriving July 2015' an `<h2>` or something else (which should automatically give you the line break)